specification/columns/availabilityzone.md

```diff
@@ -1,8 +1,8 @@
 # Availability Zone
 
-[Availability Zone](#glossary:availability-zone) is a provider assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
+[Availability Zone](#glossary:availability-zone) is a provider-assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
 
-The AvailabilityZone column SHOULD be present in the billing data. This column MUST be of type String and MAY contain null values.
+The AvailabilityZone column MUST be present in the billing data when the provider supports deploying resources within an Availability Zone. This column MUST be of type String and MAY contain null values.
```

Comment:
Should this clarify that availabilityzone values should be populated / not null, if the indicated service is deployed in an availability zone?

Reply:
Is this a multi-value field?  e.g. can a service be charged in more than one availability zone?

Reply:
That's a great question.. Right now as it stands, it would just be a string value that the provider specifies. So they could find a way to convey the different AZ values. I think your question applies to both AZ and Region. Mind creating a new issue in GH about this clarification for AZ so at least we can discuss and figure out if there's a different way this needs to be handled? I'm guessing what we have now (value chosen by provider and not mandated by FOCUS) would work but I wonder if the language needs any further refinement.